packages/core/src/main/conversation/ConversationService.test.node.js

```diff
@@ -33,21 +33,17 @@ const createMessage = (conversationRepository, content) => {
   return GenericMessage.encode(customTextMessage).finish();
 };
 
-const generatePreKeyBundles = (users, devicesPerUser) => {
-  const preKeyBundles = {};
-
-  new Array(users)
-    .fill()
-    .map(() => new UUID(4).format())
-    .forEach(userId => {
-      preKeyBundles[userId] = {};
-      new Array(devicesPerUser)
-        .fill()
-        .map(() => new UUID(4).format())
-        .forEach(deviceId => (preKeyBundles[userId][deviceId] = {}));
-    });
-
-  return preKeyBundles;
+const generatePreKeyBundle = (userCount, devicesPerUser) => {
```

Comment:
Please specify return types for functions.

Reply:
That's just a mock function in JavaScript, I can't specify return types directly here.